packages/MSBot/src/msbot-clone-services.ts

```diff
@@ -367,21 +369,23 @@ async function processConfiguration(): Promise<void> {
                         }
                     }
                     // get appSettings from botAppSite (specifically to get secret)
-                    if (botWebSite) {
-                        let botAppSettings = await runCommand(`az webapp config appsettings list -g ${args.groupName} -n ${botWebSite.name} --subscription ${args.subscriptionId}`,
+                    if (botWebSite) {                                
+                        if(args.encryption){
```

Comment:
The encryption flag check was added after the botwebsite one to keep firing the 'botsite was not found' exception since currently this is stopping the process if triggered. Should this be ignored if encryption is set to off? @tomlm